lib/pwwka/transmitter.rb

```diff
@@ -23,10 +23,17 @@ class Transmitter
 
     DEFAULT_DELAY_BY_MS = 5000
 
+    attr_reader :app_manages_connector
     attr_reader :channel_connector
 
-    def initialize
-      @channel_connector = ChannelConnector.new(connection_name: "p: #{Pwwka.configuration.app_id} #{Pwwka.configuration.process_name}".strip)
+    def initialize channel_connector: nil
```

Comment:
Could we use parenthesis around the parameter?

Reply:
https://github.com/stitchfix/pwwka/pull/98/commits/6c77929156cc7c6f153f7f8cb162e93ac32894bf